public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,78 @@
+# Setup Google Drive remote
+
+To setup Google Drive remote you should
+[create and configure Google Project](#create-and-configure-google-project) if
+you don't have one yet.
+
+## Create and Configure Google Project
+
+Since Google Drive has tight API usage quotas, creation and configuration of
+your own `Google Project` is required:
+
+1.  Log into the [Google Cloud Platform](https://console.developers.google.com)
+    account.
+2.  Create `New Project` or select available one.
+3.  Click `ENABLE APIS AND SERVICES` and search for `drive` to enable
+    `Google Drive API` from search results.
+4.  Navigate to
+    [All Credentials](https://console.developers.google.com/apis/credentials)
+    page and click `Create Credentials` to select `OAuth client ID`. It might
+    ask you to setup a product name on the consent screen.
+5.  Select `Other` for `Application type` and click `Create` to proceed with
+    default `Name`.
+6.  `client id` and `client secret` should be showed to you. Use them for
+    further DVC's configuration.
+
+```dvc
+$ dvc remote add myremote gdrive://root/my-dvc-root
+$ dvc remote modify myremote gdrive_client_id my_gdrive_client_id
+$ dvc remote modify myremote gdrive_client_secret gdrive_client_secret
+```
+
+⚠️ It is safe to share `client_id` and `client_secret` at least with your
+teammates or inside organization. These credentialds doesn't give access to your
+Google Drive storage. But Google limits global number of queires per each
+`client_id` and uncontrolled usage of your `cliend_id` by outer world can lead
+to reaching the allowed Google API usage limits.
+
+## Google Drive remote URL format
+
+Remote URL can contain `root` which is an alias of your topmost Google Drive
+directory. To point the remote to the path under Google Drive's `root`
+directory:
+
+```dvc
+$ dvc remote add myremote gdrive://root/path/to/dir
```

Comment:
use `-d` ... for now we've decided that it's better to have those  .... usually that what most users need